app/scripts/proactive/rest/studio-client.js

```diff
@@ -202,8 +202,27 @@ define(
             if (!localStorage['pa.session']) return;
 
             var that = this;
+	  
+            that.send_multipart_request(config.restApiUrl + "/submit", jobXml, {"sessionid": localStorage['pa.session']}, function (result) {
+                if (result.errorMessage) {
+                    that.alert("Cannot submit the job", result.errorMessage, 'error');
+                } else if (result.id) {
```

Comment:
I didn't write this part of the code however I think that it is the id of the submitted job..